infrastructure/code/backend_service.go

```diff
@@ -569,3 +579,23 @@ func (s *SnykCodeHTTPClient) SubmitAutofixFeedback(ctx context.Context, fixId st
 
 	return nil
 }
+
+func getCodeApiUrl(c *config.Config) (string, error) {
+	if !c.IsFedramp() {
+		return c.SnykCodeApi(), nil
```

Comment:
In general we are in favour of moving Code API URL logic out of the global config completely. But this would maybe be a scope creep here. It'll eventually happen, so I'm seeing this as an intermediate state, which is good for now!

Reply:
Also this logic is kind of supposed to be temporary, because AFAIK, in the future all instances will use the this new endpoint for code api. I guess having all this in the backend-service will allow us to easily remove this temporary logic.